css/main.css

```diff
@@ -117,3 +118,41 @@ button:hover {
 .mb-0 {
   margin-bottom: 0 !important;
 }
+
+.btn-close {
+  box-sizing: content-box;
+  width: 1em;
+  height: 1em;
+  padding: 0.25em 0.25em;
+  color: var(--black-global);
+  background: transparent
+    url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16' fill='%23000'%3e%3cpath d='M.293.293a1 1 0 011.414 0L8 6.586 14.293.293a1 1 0 111.414 1.414L9.414 8l6.293 6.293a1 1 0 01-1.414 1.414L8 9.414l-6.293 6.293a1 1 0 01-1.414-1.414L6.586 8 .293 1.707a1 1 0 010-1.414z'/%3e%3c/svg%3e")
+    center/1em auto no-repeat;
+  border: 0;
+  border-radius: 0.25rem;
+  opacity: 0.5;
+}
+
+.btn-close:hover {
+  color: var(--black-global);
+  text-decoration: none;
+  opacity: 0.75;
+}
+
+.btn-close:focus {
+  outline: 0;
+  box-shadow: 0 0 0 0.25rem rgba(var(--primary-color), 0.25);
+  opacity: 1;
+}
+
+.ms-2 {
+  margin-left: 0.5rem !important;
+}
+
+.push-to-right {
+  margin-right: auto !important;
```

Comment:
is it good to use soo many important?  
or is it kinda compulsory for utility classes?

Reply:
for the global class, my script would work for my demo website without the !important tags.  But in a real-world scenario, should a dev use my CL, these are common styles that may get overridden by the dev's local style sheet/ browser stylesheet. Hence added important to these common styles.